### DISCUSSION TITLE
UI Consistency Fixes

### DISCUSSION THREAD

**Discussion by andreelb** [elementor/elementor#22969] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Alignment (left center right) feature is in the content tab for headings, but in the style tab for text editors. Make it consistent.

<img width="290" alt="image" src="https://github.com/elementor/elementor/assets/138502530/dbb6fbde-4ed9-4922-bbfa-b74f1a50ec80">

<img width="292" alt="image" src="https://github.com/elementor/elementor/assets/138502530/330d81a9-ba92-40e3-a410-b881f0c227ea">

### Describe the solution you'd like

All features that are shared across different elements should be in the same panel for the different elements. 

Put alignment tools for headings in the style tab.

**Comment by hitsgee** [comment]
This has always bugged me. Sure it would take seconds to resolve by the team.
